src/lib/config/simulator.ts

```diff
@@ -15,13 +15,25 @@ export type RunningPlatform = (typeof AVAILABLE_PLATFORMS)[number];
 export const STARTING_TIMEOUT_WAIT_CYLCE = 2000;
 export const STARTING_TIMEOUT_ATTEMPTS = 120;
 
-export type AiProviders = "ollama" | "openai";
-export type AiProvidersEnvVars = "ollama" | "OPENAIKEY";
+export type AiProviders = "ollama" | "openai" | "heurist";
+export type AiProvidersEnvVars = "ollama" | "OPENAIKEY" | "HEURISTAIAPIKEY";
 export type AiProvidersConfigType = {
-  [key in AiProviders]: {name: string; envVar: AiProvidersEnvVars; cliOptionValue: string};
+  [key in AiProviders]: {name: string; envVar?: AiProvidersEnvVars; cliOptionValue: string};
 };
 
 export const AI_PROVIDERS_CONFIG: AiProvidersConfigType = {
-  ollama: {name: "Ollama (This will download and run a local instance of Llama 2)", envVar: "ollama", cliOptionValue: "ollama"},
-  openai: {name: "OpenAI (You will need to provide an OpenAI API key)", envVar: "OPENAIKEY", cliOptionValue: "openai"},
+  ollama: {
+    name: "Ollama (This will download and run a local instance of Llama 2)",
+    cliOptionValue: "ollama",
+  },
+  openai: {
+    name: "OpenAI (You will need to provide an OpenAI API key)",
+    envVar: "OPENAIKEY",
+    cliOptionValue: "openai",
+  },
+  heurist: {
+    name: "Heurist (You will need to provide an API key)",
+    envVar: "HEURISTAIAPIKEY",
+    cliOptionValue: "heurist",
+  },
```

Comment:
I think we should add something here that explains that you can get free credits 

> the automated invite code system is ready https://dev-api-form.heurist.ai/ (accessible from heurist.ai homepage) you can try it out and fill in "genlayer" in the last question

Reply:
what about:
- name: "Heurist (You will need to provide an API key which you can get for free at https://dev-api-form.heurist.ai by entering the referral code: genlayer)"

Reply:
> Heurist (You will need to provide an API key. Get free API credits at https://dev-api-form.heurist.ai/ with referral code: "genlayer"):